internal/elasticsearch/client.go

```diff
@@ -0,0 +1,36 @@
+package elasticsearch
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/stack"
+	"github.com/elastic/go-elasticsearch/v7"
+)
+
+// Client method creates new instance of the Elasticsearch client.
+func Client() (*elasticsearch.Client, error) {
+	host := os.Getenv(stack.ElasticsearchHostEnv)
+	if host == "" {
+		return nil, undefinedEnvError(stack.ElasticsearchHostEnv)
+	}
+
+	username := os.Getenv(stack.ElasticsearchUsernameEnv)
+	password := os.Getenv(stack.ElasticsearchPasswordEnv)
+
+	client, err := elasticsearch.NewClient(elasticsearch.Config{
+		Addresses: []string{host},
+		Username:  username,
+		Password:  password,
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "creating Elasticsearch client failed")
+	}
+	return client, nil
+}
+
+func undefinedEnvError(envName string) error {
```

Comment:
Nit: this function seems to something that would be useful generally when dealing with required environment variables, not something specific to the Elasticsearch client. So maybe consider moving it out of this file/package?

Reply:
It looks like an ordinary helper function, but I wonder what would be the best place. `stack.UndefinedEnvError` ?

Reply:
Yeah, not sure of the best place, looking at what we have at the moment. Let's just keep it here for now. We can extract it when we need it in another place. Maybe by then there will be a suitable package or we can just stick it in `stack` at the time.

Reply:
I adjusted all `fmt.Println` calls, but I think we need to change a bit the logger behavior. Take a look at this:

```
elastic-package test pipeline
Test case: test-access-event.json
2020/08/25 08:46:08  WARN Expected results are different from current ones:
2020/08/25 08:46:08  WARN  {
     "expected": [
         {
             "@timestamp": "2016-12-07T10:04:37.000Z",
```

The verbose logging is disabled, but it prints warnings. This is because the verbose logging is in fact a debug mode check. Should I change this condition to disable/enable all logger calls? In this case it would be:

```
elastic-package test pipeline
Test case: test-access-event.json
Test case: test-access-raw.log
Error: error running package pipeline tests: at least one test case failed
```

(actually you don't know which one failed)